ocaml/graph/go

```diff
@@ -1,13 +1,17 @@
 #!/bin/sh
 
+set -e
+
+GRAPH="../../graph.native -h $1 -u $2 -pw $3"
 DOTARGS="-Tpng -Ktwopi -Goverlap=false"
-GRAPH=./graph
 
 ${GRAPH} VM VBD VDI SR > vm-disks.dot
 ${GRAPH} VM VIF network > vm-network.dot
 ${GRAPH} host PIF network > host-network.dot
 
-for i in vm-disks.dot vm-network.dot host-network.dot
+for i in vm-disks vm-network host-network
 do
-  dot ${DOTARGS} $i > $i.png
+  dot ${DOTARGS} $i.dot > $i.png
 done
+
+set +e
```

Comment:
Oh, thanks for the explanation, didn't think of that use case. Would the `set -u` provide even more safety here to error out if we use an undefined variable?